src/store/slices/webrtcSlice.tsx

```diff
@@ -37,6 +39,9 @@ const webrtcSlice = createSlice({
 		setRTCStatsOptions: ((state, action: PayloadAction<RTCStatsOptions>) => {
 			state.rtcStatsOptions = action.payload;
 		}),
+		setClientMonitorConfig: ((state, action: PayloadAction<ClientMonitorConfig>) => {
```

Comment:
No biggie, but this state action is never used, do you need to store the config in the store at a later point?